### PR TITLE
Feature interaction time count of views

### PR DESCRIPTION
**Pull request by maksim-luzin**

### CODE REVIEW

File: backend/src/services/project/project.ts [hunk 1]
```diff
@@ -95,4 +105,26 @@ export default class ProjectService {
...
+  public async updateViewsAndInteractionTime({ id, interactionTime }:UpdateInteractionTimeQuery) {
+    try {
+      const { totalInteractionTime, totalViews } = await this.#projectRepository.getViewsAndInteractionTimeById(id);
+      const response = await this.#projectRepository.updateViewsAndInteractionTimeById(
+        id,
+        {
+          totalViews: !totalViews ? 1 : totalViews + 1,
+          totalInteractionTime: !totalInteractionTime ? interactionTime : totalInteractionTime + interactionTime
+        }
+      ) as unknown as UpdateViewsAndInteractionTime;
```
Comment (yevhenii-moroziuk) [inline backend/src/services/project/project.ts]: Do not use "as unknown as UpdateViewsAndInteractionTime"

File: frontend/src/components/Card/index.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/require-default-props */
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Card/index.tsx]: Seems that some of eslint dependencies have been updated (or the rule was removed), Let's disable this rule in config.

File: frontend/src/helpers/http.ts [hunk 3]
```diff
@@ -49,7 +56,7 @@ const getUrl = (method: HttpMethod, { url, params, config }: RequestArgs): strin
     return url;
   }
 
-  const fullUrl = `${env.server.url}/${url}`;
+  const fullUrl = url;
```
Comment (yevhenii-moroziuk) [inline frontend/src/helpers/http.ts]: merge develop, please

File: frontend/src/services/project.ts [hunk 4]
```diff
@@ -15,7 +15,7 @@ export const createProject = async (body: any) => {
 
 export const getProject = (
   params: GetProjectProps
-): Promise<Project> => api.get({ url: `projects/${params.id}`, params });
+): Promise<Project> => api.get({ url: params.id, params });
```
Comment (yevhenii-moroziuk) [inline frontend/src/services/project.ts]: it should work in dev

File: frontend/src/services/projects.ts [hunk 5]
```diff
@@ -41,3 +41,11 @@ export const getPopularAndRecommendedProjects = async () => {
   } = await api.get({ url: 'projects/popular-and-recommended' });
   return projects;
 };
+
+export const updateViewsAndInteraction = ({ id = '', ...params }) => {
+  if (!id || typeof params === 'undefined') throw Error();
```
Comment (yevhenii-moroziuk) [inline frontend/src/services/projects.ts]: do we need this check here?

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: backend/src/services/project/project.ts (hunk 1 above)
Comment (maksim-luzin) [reply backend/src/services/project/project.ts]: I will delete it.

File: frontend/src/services/project.ts (hunk 4 above)
Comment (maksim-luzin) [reply frontend/src/services/project.ts]: I will replace it.

File: frontend/src/services/projects.ts (hunk 5 above)
Comment (maksim-luzin) [reply frontend/src/services/projects.ts]: I will deleted it.

File: frontend/src/components/Card/index.tsx (hunk 2 above)
Comment (maksim-luzin) [reply frontend/src/components/Card/index.tsx]: I added `"react/require-default-props": "off"`

Comment (maksim-luzin) [reply frontend/src/components/Card/index.tsx]: And I will delete it.

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**